kura/org.eclipse.kura.web2/META-INF/MANIFEST.MF

```diff
@@ -55,11 +55,11 @@ Import-Package: com.eclipsesource.json;version="0.9.5",
  org.eclipse.kura.executor;version="[1.0,2.0)",
  org.eclipse.kura.internal.wire.asset;version="[1.0,2.0)",
  org.eclipse.kura.marshalling;version="[1.0,2.0)",
- org.eclipse.kura.net;version="[2.0,3.0)",
+ org.eclipse.kura.net;version="[2.1,3.0)",
  org.eclipse.kura.net.dhcp;version="[1.0,2.0)",
  org.eclipse.kura.net.firewall;version="[1.0,2.0)",
- org.eclipse.kura.net.modem;version="[2.1,3.0)",
- org.eclipse.kura.net.wifi;version="[2.0,3.0)",
+ org.eclipse.kura.net.modem;version="[2.0,3.0)",
+ org.eclipse.kura.net.wifi;version="[2.1,2.2)",
```

Comment:
Why the range here is strict? I believe the UI is not implementing but only using the wifi APIs.

Reply:
OK. I will extend to 3.0.